### PR TITLE
test: set up utils folder

### PR DESCRIPTION
**Pull request by VictorChao996**
### Description:

- Notion link: https://www.notion.so/SLD-9-Test-complete-utils-folder-c1e53c113c6f4e7192f41f1141a1badc?pvs=4
- add utils folder

### Changes:
- add mockData
- add customStatusCode
- add customStatusCodeMessage

### Screenshots (optional)
image

### CODE REVIEW

File: test/utils/customStatusCodeMessage.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default {
+    queryRequiredErrorMessage: "001: 未帶必要欄位、格式不符",
```
Comment (5-michelle) [inline test/utils/customStatusCodeMessage.js:2]: 刪除格式不符

File: test/utils/customStatusCodeMessage.js
Comment (5-michelle) [inline test/utils/customStatusCodeMessage.js:1]: 刪除在 message 中的狀態碼

File: test/utils/customStatusCodeMessage.js (hunk 1 above)
Comment (VictorChao996) [reply test/utils/customStatusCodeMessage.js]: 完成

**Review by 5-michelle — APPROVED** [review]
LGTM!

**Pull request merged**